tests/e2e/functional_test.go

```diff
@@ -524,6 +541,56 @@ var _ = Describe("Functional e2e", Serial, func() {
 
 	})
 
+	It("Should update the child ISBService with an excluded field", func() {
+
+		// new ISBService spec
+		rawSpec, err := json.Marshal(ISBServiceSpecExcludedField)
+		Expect(err).ShouldNot(HaveOccurred())
+
+		updateISBServiceRolloutInK8S(isbServiceRolloutName, func(rollout apiv1.ISBServiceRollout) (apiv1.ISBServiceRollout, error) {
+			rollout.Spec.InterStepBufferService.Spec.Raw = rawSpec
+			return rollout, nil
+		})
+
+		if dataLossPrevention == "true" {
+
+			document("Verify that in-progress-strategy gets set to NoOp")
+			verifyInProgressStrategyISBService(Namespace, isbServiceRolloutName, apiv1.UpgradeStrategyNoOp)
+			verifyInProgressStrategy(pipelineRolloutName, apiv1.UpgradeStrategyNoOp)
```

Comment:
hmm, just thinking that these may have no effect, as they may have already been "NoOp"

Reply:
This is true, is there no chance it could be set to PauseAndDrain incorrectly? If not, can remove these checks.

Reply:
or maybe you can group it with the same "consistently" code I mention below?

Reply:
I just realized the two statements above `verifyInProgressStrategyISBService` and `verifyInProgressStrategy` are `Eventually` statements themselves - we want to avoid those being within a `Consistently`.